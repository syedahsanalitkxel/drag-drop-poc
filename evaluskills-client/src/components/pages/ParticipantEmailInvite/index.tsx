import React from 'react';
import GuestTemplate from '../../templates/GuestTemplate';

const ParticipantHome = () => {
  return (
    <GuestTemplate>
      <div className="wrapper wrapper-content animated fadeInRight">
        <div className="invite-container">
          <span className="invite-title">Hey Jasmine, you’re invited to participate in the</span>
          <h1 className="font-bold mt-1 mb-4">360° Leadership Assessment</h1>
          <a href="#" className="btn btn-primary font-size-20 font-bold mb-4">
            Begin Here
            <img src="img/arrow.svg" alt="arrow" />
          </a>
        </div>
      </div>
    </GuestTemplate>
  );
};
export default ParticipantHome;
