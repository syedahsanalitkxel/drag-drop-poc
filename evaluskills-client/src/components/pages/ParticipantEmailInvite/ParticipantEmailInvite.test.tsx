import { shallow } from 'enzyme';
import ParticipantEmailInvite from './index';

describe('ParticipantEmailInvite', () => {
  const PrtEmailInvite = shallow(<ParticipantEmailInvite />);
  it('it renders correctly', () => {
    expect(PrtEmailInvite).toMatchSnapshot();
  });
});
