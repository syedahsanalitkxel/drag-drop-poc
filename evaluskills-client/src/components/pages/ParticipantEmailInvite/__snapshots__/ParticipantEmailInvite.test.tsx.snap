// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<GuestTemplate>
  <div
    className="wrapper wrapper-content animated fadeInRight"
  >
    <div
      className="invite-container"
    >
      <span
        className="invite-title"
      >
        Hey Jasmine, you’re invited to participate in the
      </span>
      <h1
        className="font-bold mt-1 mb-4"
      >
        360° Leadership Assessment
      </h1>
      <a
        className="btn btn-primary font-size-20 font-bold mb-4"
        href="#"
      >
        Begin Here
        <img
          alt="arrow"
          src="img/arrow.svg"
        />
      </a>
    </div>
  </div>
</GuestTemplate>
`;
